config/locales/en.yml

```diff
@@ -94,8 +94,8 @@ en:
     less_than: *default_message
     greater_than_or_equal_to: *default_message
   npq_application:
-    missing_npq_application: "The property '#/npq_application' must be present"
-    has_another_accepted_application: The participant already has an accepted application for this course
+    missing_npq_application: "The '#/npq_application' is missing from your request. Check details and try again."
```

Comment:
I think this should be "The entered '#/npq_application'" (to stay consistent with previous changes).